<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vo Minh Thanh</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTitleMain">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#55aaff;&quot;&gt;TRACKING TASKS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxListofTasks">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>181</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>List of Tasks:</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxDetailofTasks">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Details of Tasks</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QDateEdit" name="dateEditDeadline"/>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QLabel" name="labelContent">
        <property name="text">
         <string>Content:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelTitle">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Time:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelDeadline">
        <property name="text">
         <string>Deadline:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTimeEdit" name="timeEditTimeDeadline"/>
      </item>
      <item row="1" column="1" rowspan="2" colspan="2">
       <widget class="QLineEdit" name="lineEditContent">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditTitle"/>
      </item>
      <item row="5" column="1">
       <widget class="QRadioButton" name="radioButtonFinished">
        <property name="text">
         <string>Finished</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QRadioButton" name="radioNotFinished">
        <property name="text">
         <string>Not Finished</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxActions">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Actions:</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonNew">
        <property name="text">
         <string>New</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>image/ic_add.png</normaloff>image/ic_add.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSave">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>image/ic_save.png</normaloff>image/ic_save.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>image/ic_delete.png</normaloff>image/ic_delete.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
